### PR TITLE
Fix exercise's link

### PR DESCRIPTION
**Pull request by Limm-jk**
I corrected the changes to the exercise I worked on.

### CODE REVIEW

**Comment by BethanyG** [comment]
Hi @Limm-jk!  You need to run `prettier` or the `/format` command on this PR once you've made your change.  😄   Otherwise, the CI test will fail.

File: languages/python/exercises/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
...
-| [`numbers-I`](https://github.com/exercism/v3/tree/master/languages/python/concepts/numbers)             | Currency Exchange/Chandler's Conversion Confusion (PR in process)                                                                      | `numbers`, `ints`, `floats`, Arithmetic | [Issue #2208](https://github.com/exercism/v3/issues/2208)                                                                                | `basics`                                                                                                                 |
+| [`numbers-I`](https://github.com/exercism/v3/tree/master/languages/python/concepts/numbers)             | [Currency Exchange](https://github.com/exercism/v3/tree/master/languages/python/exercises/concept/currency-exchange)/Chandler's Conversion Confusion (PR in process)                                                                      | `numbers`, `ints`, `floats`, Arithmetic | [Issue #2208](https://github.com/exercism/v3/issues/2208)                                                                                | `basics`                                                                                                                 |
```
Comment (BethanyG) [inline languages/python/exercises/README.md]:
```suggestion
| [`numbers`](https://github.com/exercism/v3/tree/master/languages/python/concepts/numbers)             | [Currency Exchange](https://github.com/exercism/v3/tree/master/languages/python/exercises/concept/currency-exchange)                                                                    | `numbers`, `ints`, `floats`, arithmetic | [numbers](https://github.com/exercism/v3/blob/master/languages/python/exercises/concept/currency-exchange/.meta/design)                                                                                | `basics`                                                                                                                 |
```

**Review by BethanyG — CHANGES_REQUESTED** [review]

**Comment by BethanyG** [comment]
oof. I think that the bot and I crossed wires.  See my comment above.  Unfortunately, I don't think it will allow you or me to auto-commit that one.  Let me try to make the suggestion again.

File: languages/python/exercises/README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+| `none`                                                                                                  | [Restaurant Rozalynn](https://github.com/exercism/v3/tree/master/languages/python/exercises/concept/restaurant-rozalynn)                                             | `None`                                  | [none](https://github.com/exercism/v3/blob/master/languages/python/exercises/concept/restaurant-rozalynn/.meta/design.md)                | `bools`, `conditionals`, `functions-I`, `dict-methods`, `list-methods`, `loops/iteration`                                |
+| [`numbers-I`](https://github.com/exercism/v3/tree/master/languages/python/concepts/numbers)             | [Currency Exchange](https://github.com/exercism/v3/tree/master/languages/python/exercises/concept/currency-exchange)/Chandler's Conversion Confusion (PR in process) | `numbers`, `ints`, `floats`, Arithmetic | [Issue #2208](https://github.com/exercism/v3/issues/2208)                                                                                | `basics`                                                                                                                 |
```
Comment (BethanyG) [inline languages/python/exercises/README.md:15]:
```suggestion
| [`numbers`](https://github.com/exercism/v3/tree/master/languages/python/concepts/numbers)             | [Currency Exchange](https://github.com/exercism/v3/tree/master/languages/python/exercises/concept/currency-exchange)| `numbers`, `ints`, `floats`, arithmetic | [numbers](https://github.com/exercism/v3/blob/master/languages/python/exercises/concept/currency-exchange/.meta/design)                                                                                | `basics`                                                                                                                 |
```

**Review by BethanyG — APPROVED** [review]

**Comment by BethanyG** [comment]
_**sigh**_

/format

**Comment by Limm-jk** [comment]
Was there a problem? 
Was there a problem with me? 
If you let me know, I will apply it to future contributions.

**Comment by BethanyG** [comment]
No!  I just noticed that I needed to change the name of the _concept_ to just `numbers`.  I also needed to change the link to the design file from the issue to the `currency-exchange/design.md` file.  And since I made those suggestions/changes right as you were running `/format`, we sort of stepped on each other.  So - **it was my fault!**.  Apologies for that.

**Comment by BethanyG** [comment]
running `/format` in the repo can sometimes choke as well.  If your PR is coming from a fork, you have to close the PR and then re-open it to get the CI to run again - which is _totally fine_ (although it often doesn't work for me, and I usually have to try twice 😀 ).  

You can also set up prettier (the auto-formatter we are using) to run in your code editor/IDE, if that works better:  [setting up prettier](https://prettier.io/docs/en/editors.html).  I often end up running prettier in both places (locally and in the repo) - because I am always forgetting. 😝

**Comment by Limm-jk** [comment]
hahaha
I'll keep that in mind and apply it between activities from now on.
Thank you!

**Pull request merged**